cmd/satellite/main.go

```diff
@@ -101,15 +113,17 @@ func init() {
 	rootCmd.AddCommand(setupCmd)
 	rootCmd.AddCommand(diagCmd)
 	rootCmd.AddCommand(qdiagCmd)
+	rootCmd.AddCommand(paymentsCmd)
```

Comment:
let's create a top-level command `report` or `reports`, we'll likely have more of them not just on `payments`.